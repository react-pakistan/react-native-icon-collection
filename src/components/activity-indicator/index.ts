export {
  ActivityIndicator,
  ActivityIndicatorSize,
} from './activity-indicator';
