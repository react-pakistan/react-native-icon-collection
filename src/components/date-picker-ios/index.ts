export { DatePickerIOS } from './date-picker-ios';
