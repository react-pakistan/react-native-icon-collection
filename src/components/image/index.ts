export {
  IImageProps,
  Image,
  ResizeMethod,
  ResizeMode,
} from './image';
