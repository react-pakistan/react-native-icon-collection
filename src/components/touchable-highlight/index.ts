export { TouchableHighlight } from './touchable-highlight';
