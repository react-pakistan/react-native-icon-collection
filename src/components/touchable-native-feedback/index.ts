export { TouchableNativeFeedback } from './touchable-native-feedback';
