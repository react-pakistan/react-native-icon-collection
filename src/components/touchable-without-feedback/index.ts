export { TouchableWithoutFeedback } from './touchable-without-feedback';
