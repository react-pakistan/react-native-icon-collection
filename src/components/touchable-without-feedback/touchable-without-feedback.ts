import styled from 'styled-components/native';

export const TouchableWithoutFeedback = styled.TouchableWithoutFeedback``;
