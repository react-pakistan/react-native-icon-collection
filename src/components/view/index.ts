export { View, IView } from './view';
