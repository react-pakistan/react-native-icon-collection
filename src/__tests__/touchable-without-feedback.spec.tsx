import React from 'react';
import { Text } from 'react-native';
import renderer from 'react-test-renderer';
import { TouchableWithoutFeedback } from '../components/touchable-without-feedback';
// import {
// } from '../story/view.story';

describe('Testing TouchableWithoutFeedback component', () => {
  it('matches TouchableWithoutFeedback snapshot', () => {
    const wrapper = renderer.create(
      <TouchableWithoutFeedback>
        <Text>
          Hello World
        </Text>
      </TouchableWithoutFeedback>,
    );
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
