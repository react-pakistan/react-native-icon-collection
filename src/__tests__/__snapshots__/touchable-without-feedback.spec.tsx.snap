// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing TouchableWithoutFeedback component matches TouchableWithoutFeedback snapshot 1`] = `
<Text
  accessible={true}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
>
  Hello World
</Text>
`;
